server/common/HatoholArmPluginInterface.cc

```diff
@@ -292,6 +297,11 @@ void HatoholArmPluginInterface::send(
 	m_impl->sender.send(request);
 }
 
+bool HatoholArmPluginInterface::getConnectStat(void)
```

Comment:
Thank you for comments and sorry for late.
I will similarly modified(it is easy to understand.).